src/session_impl.cpp

```diff
@@ -5086,19 +5093,19 @@ namespace aux {
 					torrent_ptr->set_url(params.url);
 				if (!params.source_feed_url.empty() && torrent_ptr->source_feed_url().empty())
 					torrent_ptr->set_source_feed_url(params.source_feed_url);
-				return torrent_ptr;
+				return std::make_pair(torrent_ptr, false);
```

Comment:
does this mean...
`if ((params.flags & add_torrent_params::flag_duplicate_is_error) == 0)`

that duplicates aren't errors?

If so, is it ok to put `false` in the second element of the pair?